java-generator/README.md

```diff
@@ -6,6 +6,26 @@ This is an early preview of a Java generator from CRD and
 there might still be issues in the generated sources.
 Please give it a try and report any issue you might find.
 
+### Usage
+
+The `java-generator` is distributed as a tiny CLI application, you can obtain it by running:
+
+```
+mvn clean package -f cli/pom.xml
+```
+
+The produced executable will be `cli/target/java-gen` and it provides basic configuration options:
+
+```bash
+Usage: GenerateJavaSources [-hV] -crd=<crdFile> -dest=<destFolder>
```

Comment:
Nitpicking but usually CLI tools don't have capital letters in their names :)

Reply:
the actual name is `java-gen` let me see how to tune it 😄

Reply:
Ah yes, sorry  but then the usage message is confusing :)

Reply:
fixed